suite-native/config/src/enabledNetworks.ts

```diff
@@ -1,3 +1,32 @@
-import { networks, NetworkSymbol } from '@suite-common/wallet-config';
+import {
+    networks,
+    NetworkSymbol,
+    Network,
+    getTestnets,
+    getMainnets,
+} from '@suite-common/wallet-config';
+
+export const mainnetsOrder: NetworkSymbol[] = [
+    'btc',
+    'eth',
+    'ltc',
+    'doge',
+    'etc',
+    'ada',
+    'bch',
+    'xrp',
+    'dash',
+    'zec',
+    'btg',
+    'vtc',
+    'nmc',
+    'dgb',
```

Comment:
it is ordered as requested in the issue description. I took a look on the data and ordered the coins according usage of them in Desktop Suite. We can't 100% rely on the data because we track max 50% of users (only those who gave us consent) but at least something.

Reply:
comment added e21e5713ac754d22544aedeb5ae901093cf708a7